DocumentedExamples/NearestNeighbourDistance.ipynb

```diff
@@ -0,0 +1,1730 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    distances, indices = ball_tree.query(grid_coords, return_distance = True)
</pre><p>here the <code>distances</code> are not distances but angles on the sphere, right? shall we call them angles?</p><p> </p><p>then below we compute distances via something like</p><p> </p><pre class="ql-syntax" spellcheck="false">R_earth = 6371 # km

distances = R_earth * angles
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>